Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: all
+
+all: publish
+
+VERSION ?= $(shell echo "$${GITHUB_REF_NAME:-`grep VERSION lib/devextreme/rails/version.rb | sed -e 's/VERSION =//' -e 's/[ "]//g'`}")
```

Comment:
This uses the tag associated with the release as the version. If that is blank, then it uses the version in the version.rb file.

All of this can be overridden when calling make:
```
$ VERSION=123.123.123.123 make publish
```
will result in the version being 123.123.123.123